kotlinpoet/src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -428,6 +428,11 @@ class CodeBlock private constructor(
       formatParts += "⇤"
     }
 
+    fun clear() = apply {
```

Comment:
This could land separately. Also needs a test.

Reply:
Good point @742